NewEvaluation.css

```diff
@@ -68,3 +92,107 @@ body{
   	width:100%;  background-color: #dfdfdf; text-align: center; vertical-align: center; padding-bottom: 1em;
   	color:#828384;
 }
+
+
+.legend-box legend{
+	color:#03a8f3;
+	font-size: 18px;
+	font-weight: 500;
+	padding-left: 1em;
+	padding-right: 1em;
+}
+
+.legend-box li{
+	float:left;
+	width: 33%;
+	padding-bottom: 10px;
+}
+.legend-box select{
+	width: 40%;
+	padding: 3px;
+}
+.legend-box label{
+	display: inline-block;
+	width: 49%;
+}
+
+
+
+
+#butonu{
+	color:red;
+}
+
+#extra{
```

Comment:
One of the first things people tend to do when writing CSS is to (ab)use ID selectors. While this is not necessarily an awful thing to do, it's usually one of the last things you want to do, because styling against IDs is notoriously difficult to override, due to a little thing called specificity. A best practice in this sense is to always prefer class names for styling, wherever possible / applicable.